comp/livestream/hardware.yaml

```diff
@@ -0,0 +1,21 @@
+summary: Organise hardware to run livestream
+
+priority: major
+
+component: Competition
+
+milestone: $SRYYYY Competition Prep
+
+description: >-
+  This ticket relates to providing the actual machine that will be
+  used for the livestream.
+
+  Requirements:
+  - Video mixer PC (With dedicated GPU, capable of running OBS)
+  - 2× HDMI capture card
+  - 2× mixers
```

Comment:
Can't really think why this needs 2 but we can always change this later.

Reply:
This was copied from the previous ticket. @Tyler-Ward may know more.

Reply:
I'm not sure where two mixers have come from. may have been munged from the two streamdecks used a couple of years ago.

A single audio mixer to mix the voice for the livestream would be good, an alternative option would be to do the mixing in OBS which would require an audio interface for the livestream presenters and the match presenters.